src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
Moving `.EXAMPLE` to `defaults` sounds good to me. Maybe also update the PR to show that.

I'm not too familiar with war overlays. Is there any documentation you can recommend on why that would be better than using environment variables?